CRITICAL - 2025-04-25 01:18:33 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/adminlte.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/adminlte.php')
 2 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 4 APPPATH\Controllers\Dashboard.php(12): view('dashboard', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 01:21:19 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/adminlte.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/adminlte.php')
 2 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 4 APPPATH\Controllers\Dashboard.php(12): view('dashboard', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 01:24:49 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/adminlte.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/adminlte.php')
 2 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 4 APPPATH\Controllers\Dashboard.php(12): view('dashboard', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 01:26:40 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/adminlte.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/adminlte.php')
 2 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 4 APPPATH\Controllers\Dashboard.php(12): view('dashboard', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 01:27:59 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/adminlte.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/adminlte.php')
 2 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 4 APPPATH\Controllers\Dashboard.php(12): view('dashboard', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 01:28:01 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/adminlte.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/adminlte.php')
 2 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 3 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 4 APPPATH\Controllers\Dashboard.php(12): view('dashboard', [...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 01:29:02 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/footer.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/footer.php')
 2 SYSTEMPATH\View\View.php(475): CodeIgniter\View\View->render('templates/footer', null, true)
 3 APPPATH\Views\templates\adminlte.php(17): CodeIgniter\View\View->include('templates/footer')
 4 SYSTEMPATH\View\View.php(224): include('D:\\xampp\\htdocs\\pelmal\\app\\Views\\templates\\adminlte.php')
 5 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 7 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 8 APPPATH\Controllers\Dashboard.php(12): view('dashboard', [...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 01:31:25 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/footer.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/footer.php')
 2 SYSTEMPATH\View\View.php(475): CodeIgniter\View\View->render('templates/footer', null, true)
 3 APPPATH\Views\templates\adminlte.php(17): CodeIgniter\View\View->include('templates/footer')
 4 SYSTEMPATH\View\View.php(224): include('D:\\xampp\\htdocs\\pelmal\\app\\Views\\templates\\adminlte.php')
 5 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 7 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('minimal', [], true)
 8 APPPATH\Controllers\Dashboard.php(14): view('minimal')
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 02:24:48 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "templates/parts/scripts.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('templates/parts/scripts.php')
 2 SYSTEMPATH\View\View.php(475): CodeIgniter\View\View->render('templates/parts/scripts', null, true)
 3 APPPATH\Views\templates\adminlte.php(15): CodeIgniter\View\View->include('templates/parts/scripts')
 4 SYSTEMPATH\View\View.php(224): include('D:\\xampp\\htdocs\\pelmal\\app\\Views\\templates\\adminlte.php')
 5 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\View\View.php(240): CodeIgniter\View\View->render('templates/adminlte', [], true)
 7 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 8 APPPATH\Controllers\Dashboard.php(11): view('dashboard', [...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-25 04:06:12 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in D:\xampp\htdocs\pelmal\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 D:\xampp\htdocs\pelmal\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 D:\xampp\htdocs\pelmal\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\pelmal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\pelmal\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 D:\xampp\htdocs\pelmal\system\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 D:\xampp\htdocs\pelmal\app\Controllers\Antrian.php(112): CodeIgniter\Model->countAllResults()
#6 D:\xampp\htdocs\pelmal\app\Controllers\Antrian.php(21): App\Controllers\Antrian->getNomorAntrianHariIni()
#7 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(933): App\Controllers\Antrian->index()
#8 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Antrian))
#9 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\pelmal\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\pelmal\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\pelmal\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in D:\xampp\htdocs\pelmal\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 D:\xampp\htdocs\pelmal\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\pelmal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\pelmal\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 D:\xampp\htdocs\pelmal\system\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#4 D:\xampp\htdocs\pelmal\app\Controllers\Antrian.php(112): CodeIgniter\Model->countAllResults()
#5 D:\xampp\htdocs\pelmal\app\Controllers\Antrian.php(21): App\Controllers\Antrian->getNomorAntrianHariIni()
#6 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(933): App\Controllers\Antrian->index()
#7 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Antrian))
#8 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\pelmal\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\pelmal\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\pelmal\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-25 04:06:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: antrian]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `antrian`
WHERE `tanggal` = :tanggal:', [...], false)
 3 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 4 APPPATH\Controllers\Antrian.php(112): CodeIgniter\Model->countAllResults()
 5 APPPATH\Controllers\Antrian.php(21): App\Controllers\Antrian->getNomorAntrianHariIni()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Antrian->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Antrian))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-04-25 04:07:07 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in D:\xampp\htdocs\pelmal\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 D:\xampp\htdocs\pelmal\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 D:\xampp\htdocs\pelmal\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\pelmal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\pelmal\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 D:\xampp\htdocs\pelmal\system\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 D:\xampp\htdocs\pelmal\app\Controllers\Antrian.php(112): CodeIgniter\Model->countAllResults()
#6 D:\xampp\htdocs\pelmal\app\Controllers\Antrian.php(21): App\Controllers\Antrian->getNomorAntrianHariIni()
#7 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(933): App\Controllers\Antrian->index()
#8 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Antrian))
#9 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\pelmal\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\pelmal\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\pelmal\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in D:\xampp\htdocs\pelmal\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 D:\xampp\htdocs\pelmal\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\pelmal\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\pelmal\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 D:\xampp\htdocs\pelmal\system\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#4 D:\xampp\htdocs\pelmal\app\Controllers\Antrian.php(112): CodeIgniter\Model->countAllResults()
#5 D:\xampp\htdocs\pelmal\app\Controllers\Antrian.php(21): App\Controllers\Antrian->getNomorAntrianHariIni()
#6 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(933): App\Controllers\Antrian->index()
#7 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Antrian))
#8 D:\xampp\htdocs\pelmal\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\pelmal\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\pelmal\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\pelmal\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-25 04:07:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: GET, Route: antrian]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `antrian`
WHERE `tanggal` = :tanggal:', [...], false)
 3 SYSTEMPATH\Model.php(676): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 4 APPPATH\Controllers\Antrian.php(112): CodeIgniter\Model->countAllResults()
 5 APPPATH\Controllers\Antrian.php(21): App\Controllers\Antrian->getNomorAntrianHariIni()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Antrian->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Antrian))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 06:18:58 --> CodeIgniter\Database\Exceptions\DatabaseException: You must set the database table to be used with your query.
[Method: GET, Route: dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 941.
 1 SYSTEMPATH\Model.php(714): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(920): CodeIgniter\Model->builder()
 3 APPPATH\Models\KepuasanModel.php(11): CodeIgniter\Model->__call('where', [...])
 4 APPPATH\Controllers\Dashboard.php(19): App\Models\KepuasanModel->countRating(1)
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-25 06:56:40 --> ParseError: syntax error, unexpected variable "$data", expecting "function" or "const"
[Method: GET, Route: kepuasan]
in APPPATH\Controllers\Kepuasan.php on line 8.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('D:\\xampp\\htdocs\\pelmal\\app\\Controllers\\Kepuasan.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Kepuasan')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Kepuasan')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Kepuasan', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
